static/app/views/replays/detail/replayMetaData.tsx

```diff
@@ -98,4 +141,15 @@ const KeyMetricData = styled('div')`
   line-height: ${p => p.theme.text.lineHeightBody};
 `;
 
+const ErrorTag = styled(Tag)`
+  ${Background} {
+    background: ${p => p.theme.tag.error.iconColor};
+    padding: 0 ${space(0.75)};
+
+    span {
+      color: white !important;
```

Comment:
😞

Reply:
it's always white because it's on a red background